### PR TITLE
Package dolmen.0.4

### PR DESCRIPTION
**Pull request by Gbury**
### `dolmen.0.4`
A parser library
Dolmen is a parser library. It currently targets languages used in automated theorem provers,
but may be extended ot other domains.

Dolmen provides functors that takes as arguments a representation of terms and statements,
and returns a module that can parse files (or streams of tokens) into the provided representation
of terms or statements. This is meant so that Dolmen can be used as a drop-in replacement of existing
parser, in order to factorize parsers among projects.

Additionally, Dolmen also provides a standard implementation of terms and statements that cna be
used ot instantiate its parsers.

---
* Homepage: https://github.com/Gbury/dolmen
* Source repo: git+https://github.com/Gbury/dolmen.git
* Bug tracker: https://github.com/Gbury/dolmen/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/dolmen/dolmen.0.4/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+"Dolmen is a parser library. It currently targets languages used in automated theorem provers,
+but may be extended ot other domains.
+
+Dolmen provides functors that takes as arguments a representation of terms and statements,
+and returns a module that can parse files (or streams of tokens) into the provided representation
+of terms or statements. This is meant so that Dolmen can be used as a drop-in replacement of existing
+parser, in order to factorize parsers among projects.
+
+Additionally, Dolmen also provides a standard implementation of terms and statements that cna be
+used ot instantiate its parsers."
```
Comment (avsm) [inline packages/dolmen/dolmen.0.4/opam]:
```suggestion
used to instantiate its parsers."
```

File: packages/dolmen/dolmen.0.4/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "menhir" {>= "20151005"}
+  "dune" {build}
+]
+tags: [ "parser" "tptp" "logic" "smtlib" "dimacs" ]
+homepage: "https://github.com/Gbury/dolmen"
+doc: "http://gbury.github.io/dolmen"
+bug-reports: "https://github.com/Gbury/dolmen/issues"
+dev-repo: "git+https://github.com/Gbury/dolmen.git"
+authors: "Guillaume Bury"
+synopsis: "A parser library"
```
Comment (avsm) [inline packages/dolmen/dolmen.0.4/opam]:
```suggestion
synopsis: "A parser library that targets languages used in automated theorem provers"
```

**Comment by avsm** [comment]
Thanks for the contribution! I've made a couple of suggestions to clarify the description. If you want to review those, we can merge after that.

**Comment by Gbury** [comment]
I reviewed and integrated the suggested changes, thanks a lot for the feedback !

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**